src/components/media-player/dialog-media-player-group.ts

```diff
@@ -80,6 +85,9 @@ class DialogMediaPlayerGroup extends LitElement {
             slot="actionItems"
           ></ha-icon-button>
         </ha-dialog-header>
+        ${this._error
+          ? html`<ha-alert alert-type="error">${this._error}</ha-alert>`
```

Comment:
You also import `ha-alert`

Reply:
Thanks, I've added it now. I tested it when adding it and it seems to work without. Why is that? The component just happens to be imported by something else in the tree or?

Reply:
You should add the component import always. That may be possible if you've visited a different page that loads the same ha-alert component, then it shows up. If that's not the case, you won't be able to see the ha-alert, since it doesn't know yet where the component is defined.